README.md

```diff
@@ -1,5 +1,7 @@
 # What is this?
-This is an authentication plugin for Radicale 2. It adds an LDAP authentication backend which can be used for authenticating users against an LDAP server.
+This is an authentication plugin for Radicale 3. It adds an LDAP authentication backend which can be used for authenticating users against an LDAP server.
+
+Use the `2-final` git tag for Radicale 2 support.
```

Comment:
Maybe we should reference the `2` branch as there has been an unexpected additional contribution to the Radicale 2 plugin since the creation of the `2-final` tag :) Maybe there are more to come